### PR TITLE
Bulk get query documentation

### PR DESCRIPTION
**Pull request by JKDingwall**
## Overview

Update the documentation for /{db}/_bulk_get to reference the /{db}/{docid} query parameters.  This code indicates that they share the same query parsing function:

```
 bulk_get_parse_doc_query(Req) ->
    lists:foldl(
        fun({Key, Value}, Args) ->
            ok = validate_query_param(Key),
            parse_doc_query({Key, Value}, Args)
        end,   
        #doc_query_args{},
        chttpd:qs(Req)
    ).
```

## Related Issues or Pull Requests
- https://github.com/apache/couchdb/issues/5236

## Checklist

- [ ] Code is written and works correctly
- [ ] Changes are covered by tests
- [ ] Any new configurable parameters are documented in `rel/overlay/etc/default.ini`
- [x] Documentation changes were made in the `src/docs` folder
- [ ] Documentation changes were backported (separated PR) to affected branches

### CODE REVIEW

**Comment by JKDingwall** [comment]
I'm not sure how to test the changes to make sure that they render correctly and that the `:ref:\`document endpoint <api/doc>` works as expected.

**Comment by nickva** [comment]
@JKDingwall thank you for your PR. It looks 

To check it locally, `cd src/docs` and run `./setup/.sh`. That should setup a python venv with all dependencies. 

Then run `make check` to check for basic formatting issues (line lengths, empty lines), then run `make html` to build the html docs.

To view them locally ` open build/html/index.html` that opens the browser (on macos at least) to the built docs page.

**Review by nickva — APPROVED** [review]
+1

I'll merge it after another pending PR merges.

**Comment by nickva** [comment]
The PR merged. Thank you for your contribution, @JKDingwall

**Pull request merged**
